aiohttp/web_urldispatcher.py

```diff
@@ -977,9 +977,13 @@ def register_resource(self, resource: AbstractResource) -> None:
         name = resource.name
 
         if name is not None:
+            if keyword.iskeyword(name):
```

Comment:
Should we trim whitespaces first?
```suggestion
            if keyword.iskeyword(name.strip()):
```